integration-tests/src/test/resources/ror_audit/enabled_auditing_tools/readonlyrest.yml

```diff
@@ -6,9 +6,17 @@ readonlyrest:
       - type: index
         index_template: "'audit_index'"
         serializer: "tech.beshu.ror.audit.instances.DefaultAuditLogSerializerV1"
+        serialize_all_allowed_events: false
+        fields:
+          node_name_with_static_suffix: "{ES_NODE_NAME} with suffix"
```

Comment:
It's maybe an overkill to allow that much configurability, but it basically allows to define a serializer in config, without writing any code. 

Admins will be able to define any fields they want. The values are strings, that may contain any values defined by user, combined with value placeholders provided by ROR. If only our placeholder with number-like value is used, then it is reported as number. Other values and combinations of values are strings.